source/beamline/BioXAS/BioXASMainBeamline.cpp

```diff
@@ -360,6 +360,30 @@ void BioXASMainBeamline::setupComponents()
 	if (softIn3)
 		softIn3->setTimeBeforeResetPreference(0.01);
 
+	BioXASZebraOutputControl *out1TTL = zebra_->outputControlAt(0);
```

Comment:
is it possible to encrypt these configuration inside `BioXASMainZebra`? Those configurations look like specific to the BioXASMainZebra.